src/lucky_migrator/create_table_statement.cr

```diff
@@ -118,6 +119,41 @@ class LuckyMigrator::CreateTableStatement
     end
   end
 
+  def default_value(type : String.class, default : String, name)
+    if %w[now() current_time current_timestamp].includes?(default.downcase)
+      return " DEFAULT NOW()"
+    end
+    return " DEFAULT '#{default}'"
+  end
+
+  def default_value(type : Time.class, default : (String | Time | Symbol), name)
```

Comment:
If we do just `:now` we can simplify this to just `Time | Symbol`

Reply:
Good call, much simpler